CONTRIBUTING.rst

```diff
@@ -46,8 +46,34 @@ Every issue will get a **unique issue number**.
 
 **Step 2**: Create a local branch that corresponds to the issue. There are utility scripts to help you with this::
 
-  $ ./tools/create-patch-branch <unique issue number> '<component> #<issue number>: <short_change_message>'
-  $ ./tools/create-feature-branch <unique issue number> '<component> #<issue number>: <short_change_message>'
+  $ ./tools/create-patch-branch <unique issue number> '<short_change_message> #<issue number>: '
+  $ ./tools/create-feature-branch <unique issue number> '<short_change_message> #<issue number>'
+
+The message can be possibly with
```

Comment:
Rewrite proposal:

"If you add a github-recognised [keyword](https://help.github.com/articles/closing-issues-using-keywords/) then the associated issue can be closed automatically once the pull request is merged."